jekyll/_cci2/optimization-cookbook.md

```diff
@@ -0,0 +1,215 @@
+description: "Starting point for Optimizations Cookbook"
+categories: [getting-started]
+order: 1
+---
+
+The *CircleCI Optimizations Cookbook* is a collection of individual use cases (referred to as "recipes") that provide you with detailed, step-by-step instructions on how to perform various optimization tasks using CircleCI resources. This guide, and it associated sections, will enable you to quickly and easily perform repeatable optimization tasks on the CircleCI platform.
+
+* TOC
+{:toc}
+
+## Introduction
+
+Sometimes when you are using the CircleCI platform, you may encounter unexpected lags in pipeline performance, which can negatively affect your ability to perform critical organizational functions. These performance bottlenecks can not only impact overall performance, but also cause workflow and build failures. These "hiccups" can cost you money in terms of credit usage, resources, and individual time spent reducing bottlenecks.
+
+This guide provides you with the following optimization strategies that you can utilize to minimize any potential performance bottlenecks and ensure that you are getting the best performance possible when using CircleCI:
+
+- Running jobs sequentially to prevent concurrency
+- Implementing caching strategies to optimize builds and workflows
+- Improving test performance
+
+**Note:** This guide will be updated with new optimization strategies on a continual basis, so please feel free to refer to this page for new and updated content.
+
+### Optimization Recipes
+
+The table below lists the different optimization tasks that can be performed using the CircleCI platform.
+
+Recipe | Description 
+------------|-----------
+Running Jobs Sequentially To Prevent Concurrency | This section describes how you can use the queue orb to run jobs sequentially without concurrency.
+Using Caching to Optimize Builds and Workflows |  This section describes how you can use different caching strategies in your implementations to optimize your builds and workflows.
+Improving Test Performance | This section describes a specific use case where test performance was significantly improved on the CircleCI platform.
+
+## Running Jobs Sequentially To Prevent Concurrency
+
+One of the most common tasks you may encounter when using the CircleCI platform is managing multiple jobs simultaneously to ensure your operations do not fail because of system timeouts. This becomes especially important when you have multiple contributors and committers working in the same environment. Because the CircleCI platform was designed to handle multiple tasks simultaneously without performance degradation or latency, concurrency may sometimes become an issue if there are a large number of jobs being queued, waiting for a previous job to be completed before the new job can be initiated, and the user has set the system timeout is too low. In this case, one job will be completed, and other jobs will fail due to this timeout setting.
```

Comment:
```suggestion
One of the most common tasks you may encounter when using the CircleCI platform is managing multiple jobs simultaneously to ensure your operations do not fail because of system timeouts. This becomes especially important when you have multiple contributors and committers working in the same environment. Because the CircleCI platform was designed to handle multiple tasks simultaneously without performance degradation or latency, concurrency may sometimes become an issue if there are a large number of jobs being queued, waiting for previous jobs to be completed before new jobs can be initiated, and system timeout is set too low. In this case, one job will be completed, and other jobs will fail due to this timeout setting.
```